core/templates/dev/head/pages/exploration_editor/exploration_editor.html

```diff
@@ -73,26 +73,26 @@
       </div>
       <!-- This is an ng-if, so that the preview loads the latest version of the exploration each time the tab is accessed. -->
       <div ng-if="getActiveTabName() === 'preview'">
-        {% include 'pages/exploration_editor/preview_tab/preview_tab.html' %}
+        <preview-tab></preview-tab>
       </div>
 
       <div ng-show="getActiveTabName() === 'stats'">
-        {% include 'pages/exploration_editor/statistics_tab/statistics_tab.html' %}
+        <statistics-tab></statistics-tab>
       </div>
 
       <improvements-tab ng-if="getActiveTabName() === 'improvements'">
       </improvements-tab>
 
       <div ng-show="getActiveTabName() === 'settings'">
-        {% include 'pages/exploration_editor/settings_tab/settings_tab.html' %}
+        <settings-tab current-user-is-admin="currentUserIsAdmin" current-user-is-moderator="currentUserIsModerator"></settings-tab>
```

Comment:
I had to pass these as bindings since these were in the global $scope soup and the directive needed access to it.